foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -109,6 +109,23 @@
             "includes": ["src/**/*.java", "src/**/*.properties"]
           }
         ]
+      },
+      {
+        "name": "upgrade-project-specific-image-naming-convention-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.ProjectSpecificImageNameMigration",
+        "fileSets": [
+          {
+            "includes": [
+              "*-deploy/src/main/resources/apps/*/values*.yaml",
+              "*-deploy/src/main/resources/apps/spark-worker-image/spark-worker-image.yaml",
+              "*-deploy/src/main/resources/apps/aissemble-machine-learning-training-image/aissemble-machine-learning-training-image.yaml",
+              "*-deploy/src/main/resources/apps/airflow/values-dev.yaml",
+              "*-pipelines/*/src/main/resources/apps/*-base-values.yaml",
```

Comment:
Think we can get rid of these as they'll be picked up as part of `values*.yaml`:
```
"*-deploy/src/main/resources/apps/spark-worker-image/spark-worker-image.yaml",
              "*-deploy/src/main/resources/apps/aissemble-machine-learning-training-image/aissemble-machine-learning-training-image.yaml",
              "*-deploy/src/main/resources/apps/airflow/values-dev.yaml",
```